data/models/AudioFiles.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * AudioFiles.js
+ *
+ * Data model for storing audio files.
+ * 
+ * Author: Tamara Hills
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+const dynamoose = require('dynamoose');
+
+const schema = new dynamoose.Schema({
+  item_id: {
+    type: String,
+    hashKey: true
+  },
+  uuid: {
+    type: String,
+    rangeKey: true,
+    required: true
+  },
+  type: String, // Can be intro, outro, article, mobile
```

Comment:
The inclusion of 'mobile' here is confusing since the other values are the various audio fragments. I do see the comment in AudioFileHelper that "* Mobile file has the stitched intro and the body of the file." If it makes sense to store that separately then a more explicit type name like 'intro_and_article' would be clearer.